catalog.yml

```diff
@@ -141,6 +141,7 @@
 - repo_url: https://github.com/QUIVIRR/Enhanced-Data-Sessions
   html_url: https://quivirr.github.io/Enhanced-Data-Sessions/
   preprint_citation: doi:10.31235/osf.io/eu7yw
+  journal_citation: doi:10.5278/ojs.quivirr.v1.2020.a0001
```

Comment:
Yes with 0.5.2.

> maybe falling back to Zotero with the DOI URL https://doi.org/10.5278/ojs.quivirr.v1.2020.a0001 could be a last resort attempt to retrieve DOI metadata.

I agree this is a good next step.

Reply:
I did not have trouble with this DOI with Manubot in 2021. It is only since the new year (and updating to 0.5.2) that this happened. And GitHub Actions build times are lately 3-5 times as long. No idea why... but the main processing hike is with the Build Manuscript.

Reply:
@skandilocks one of the main APIs citation metadata is getting pulled from has been having issues, see https://github.com/crosscite/content-negotiation/issues/104#issuecomment-1015311377.

One thing that will speed things up tremendously is to cache reference metadata [like](https://github.com/manubot/rootstock/blob/989fa3d1d0a36dd13f8c34ef7615f424766bc4fc/USAGE.md#reference-metadata): 

> In order to freeze all references, rather than have Manubot regenerate them during future builds, copy the `references.json` output file to `content` with a filename matching the `manual-references*.json` pattern. One tip is to embed the date `references.json` was generated into the frozen manual reference filename, like `content/manual-references-2019-06-21.json`.